pkg/document/resolvehandler/resolvehandler.go

```diff
@@ -483,13 +519,21 @@ func getEquivalentID(metadata document.Metadata) string {
 func (r *ResolveHandler) requestDiscovery(did string) {
 	logger.Infof("requesting discovery for did[%s]", did)
 
+	requestDiscoveryStartTime := time.Now()
+
+	defer r.metrics.RequestDiscoveryTime(time.Since(requestDiscoveryStartTime))
```

Comment:
you need to use fun{} in defer

Reply:
@sandrask like this https://github.com/trustbloc/orb/commit/1991dcacf9b7ec51bac6110a1390a474eca2f007#diff-a02855dc080e09ec184054aa8d77419b1d8e4b5d94b13d46def96df8868a0139R90

Reply:
done